### ISSUE TITLE
Feature request: Add "consumer" to brews

### ISSUE THREAD

**Issue by tokenstar** [graphefruit/Beanconqueror#191]
I make coffee for others, so having a way of tracking this enables lots of fun things in near and long term 

Personal analytics - "minutes without coffee" applies to me only? and number consumed per month is per user

Scores are subjective so could eventually lead to comparing preferences between individuals

**Comment by graphefruit** [comment]
Hi @tokenstar,
thanks for the request.
To be honest, I need to think about this request, because it drives into a complete new field, where I didn't think about right now or in the past.
Right now it looks like an edge-case request, because the App is and was designed to get the best shots as possible for your brews (therefore the customer is yourself or a friend e.g.).

At all, I understand the wish and the request, but there a multiple questions e.g.
I make a coffee pot (600ml) for me, and a friend, should be 2 persons be linked in the end, and how would this be splitted, 50/50? 

In my mind theres already growing an idea how it could look like. 
But I really need sometime thinking about this, I link this issue also because it drives for me into the same direction: #189 (I'll commend there also)

At all: This won't be developed in the near future.
Version 6 is already a huge load with QR-Code and roaster cooperations.
Greetings
Lars

**Comment by simons-plugins** [comment]
Yes I would love this too, and a simple hide from stats would be a simple start

**Comment by MarcinSachs** [comment]
This is a good example of a feature that becomes more interesting once data can be shared or aggregated across devices.

I’m currently experimenting with an optional server-based mode for Beanconqueror (offline-first, but with optional sync), which could make things like tracking who a brew was made for much more flexible in the future — without breaking the standalone workflow.
